### PR TITLE
form-login

### PR DESCRIPTION
**Pull request by luuthomvp**
[x] reek: failed
[o] rubocop: success
[o] rails_best_practices: success
[o] eslint: success
[o] bundle-audit: success
[o] brakeman: success
[o] scss-lint: success
[o] rspec: success
![screenshot from 2017-03-17 13 22 26]
screenshot from 2017-03-17 22 41 24

### CODE REVIEW

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          <div class="col_full nobottommargin">
+            <%= f.submit t(".login"),
+              class: "button button-3d button-green nomargin" %>
+          </div>
+      <% end %>
+
+      <div class="line line-sm"></div>
+
+      <div class="center">
+        <h4 class="login-with-custom" "><%= t ".or_login_with" %></h4>
```
Comment (nhunght) [inline app/views/devise/sessions/new.html.erb]: thừa 1 dấu " này

File: app/views/devise/sessions/_form_login.html.erb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<div class="panel panel-default divcenter pnl-login">
+  <div class="panel-body login-custom">
+    <%= form_for :user, class: "nobottommargin" do |f| %>
+        <h3 class="text-center"><%= t ".login" %></h3>
```
Comment (nhunght) [inline app/views/devise/sessions/_form_login.html.erb]: indent

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -3,30 +3,36 @@
...
-          </li>
-          <li><a href="#"><div><%= t "education.header.project" %></div></a>
-          </li>
-          <li class="mega-menu"><a href="#"><div><%= t "education.header.trainers" %></div></a>
-          </li>
-          <li class="mega-menu"><a href="#"><div><%= t "education.header.trainees" %></div></a>
-          </li>
+          <% if user_signed_in? %>
+            <li>
+            <%= link_to t(".logout"), destroy_user_session_path,
```
Comment (nhunght) [inline app/views/layouts/_header.html.erb]: indent

File: app/views/pages/home.html.erb [hunk 4]
```diff
@@ -1,4 +1,31 @@
...
+        <div class="swiper-slide dark slide-custom">
+          <div class="bg-slide-custom">
+            <img src="/assets/welcome1.jpg">
+          </div>
+        </div>
+        <div class="swiper-slide dark slide-custom">
+          <div class="video-wrap">
+            <video id="slide-video" poster="/assets/videos/explore.jpg" preload="auto" loop autoplay muted>
+              <source src='/assets/videos/explore.webm' type='video/webm' />
+              <source src='/assets/videos/explore.mp4' type='video/mp4' />
```
Comment (nhunght) [inline app/views/pages/home.html.erb]: double quote

File: app/views/pages/home.html.erb [hunk 5]
```diff
@@ -1,4 +1,31 @@
...
+            </video>
+            <div class="video-overlay" style="background-color: rgba(0,0,0,0.55);"></div>
```
Comment (Kominam) [inline app/views/pages/home.html.erb]: anh diệu k thích viết style trong view như này :dancer:

**Pull request merged**